Source/WebCore/accessibility/ios/AccessibilityObjectIOS.mm

```diff
@@ -171,6 +175,37 @@
 #endif // HAVE (INLINE_PREDICTIONS)
 }
 
+static RetainPtr<NSDictionary> unarchivedTokenForData(RetainPtr<NSData> tokenData)
+{
+    NSError *error = nil;
+    return [NSKeyedUnarchiver unarchivedObjectOfClasses:[NSSet setWithObjects:[NSDictionary class], [NSNumber class], [NSString class], nil] fromData:tokenData.get() error:&error];
+}
+
+std::span<const uint8_t> AXRemoteFrame::generateRemoteToken() const
+{
+    return span(Accessibility::newAccessibilityRemoteToken([[NSUUID UUID] UUIDString]));
+}
+
+void AXRemoteFrame::initializePlatformElementWithRemoteToken(std::span<const uint8_t> token, int processIdentifier)
+{
+    m_processIdentifier = processIdentifier;
+
+    RetainPtr nsToken = toNSData(token);
+    NSDictionary *tokenDictionary = nsToken ? unarchivedTokenForData(nsToken).get() : nullptr;
```

Comment:
Should this be `: nil` rather than `: nullptr` since this is assigning an `NSDictionary *`?

Reply:
Good point, I can change that!

Reply:
Noticed this one is marked resolved but it's still `nullptr` instead of `nil`, not sure if you got around to fixing this one yet though

Reply:
Somehow didn't staged + commit this one, sorry! Will be fixed in latest push